### ISSUE TITLE
Unable to reproduce the results for the ITOP side view human body dataset.

### ISSUE THREAD

**Issue by Shreyas-NR** [zhangboshen/A2J#59]
Hi @zhangboshen,
I'm trying to reproduce the results for the ITOP side view of the human body dataset. 
I followed all the hyperparameters settings given in the paper however, I'm not able to get the expected accuracy(mAP). I'm stuck in this step for more days now Any help is appreciated. 

The results are as below:
[train.log](https://github.com/zhangboshen/A2J/files/9668545/train.log)

RMSE
![RMSE](https://user-images.githubusercontent.com/58805235/192876802-7673dbae-2df9-49fd-873a-cf5a089b7802.png)

Loss vs epoch
![Loss vs epoch](https://user-images.githubusercontent.com/58805235/192876804-4b3264d8-d589-40f0-b570-fe3baf5b1fda.png)

Accuracy
![Accuracy](https://user-images.githubusercontent.com/58805235/192876806-22ae7085-3cb2-446a-b749-284357562bd1.png)

[ITOP_trainingCode.txt](https://github.com/zhangboshen/A2J/files/9668557/ITOP_trainingCode.txt)

Looking forward to some help!

**Comment by zhangboshen** [comment]
@Shreyas-NR 
Hi, I just checked my original training code and upload it here: https://github.com/zhangboshen/A2J/blob/master/src_train/results/ITOP_side_results.zip
I hope this could help you.
